lib/service/logger.js

```diff
@@ -14,19 +14,23 @@
 
 const isNil = require('lodash.isnil');
 
+const util = require('util');
+
 function _log(level, message) {
   if (isNil(message)) return;
   if (message instanceof Error) message = message.toString();
 
-  const logMessage = JSON.stringify({
+  const logMessage = {
     message,
     level,
     dlcId: process.env.DLC_ID,
     backingServerId: process.env.BACKING_SERVER_ID
-  });
+  };
+
+  const logMessageString = `${util.format(JSON.stringify(logMessage))}\n`;
```

Comment:
@galniv That `util.format` call was originally added to make this logger behave similarly to `console.log` but with modified output. It turns out that I'm going to add that functionality in a separate PR so I'd mistakenly left the redundant call in this branch. Removed.